grails-app/conf/Config.groovy

```diff
@@ -137,6 +137,8 @@ environments {
         grails.serverURL = "http://${localhostAddress}:8080/$appName"
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "https://catalogue-123.aodn.org.au/geonetwork"
+
+        grails.mail.port = com.icegreen.greenmail.util.ServerSetupTest.SMTP.port
```

Comment:
See https://grails.org/plugin/greenmail

Basically, enables one to test sending emails without having to rely on an external SMTP server.